docs/learn/getting-started.mdx

```diff
@@ -0,0 +1,200 @@
+    <tr>
+      <td style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+        Smart contract based profiles are the essential foundation for a better
+        web3 experience.
+      </td>
+      <td style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+        Extensible Token that move us beyond DeFi. The future of blockchain is
+        social.
+      </td>
+      <td style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+        Flexible NFTs that are extensible and directly composable with Universal
+        Profiles.
+      </td>
+    </tr>
+    <tr>
+      <td style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+        <div className="chip-container">
+          <Chip label="Visual" />
+          <Chip label="Upgradeable" />
+          <Chip label="Security" />
+          <Chip label="Gas-less" />
+          <Chip label="Smart" />
+          <Chip label="Extensible" />
+        </div>
+      </td>
+      <td>
+        <div className="chip-container">
+          <Chip label="Safer" />
+          <Chip label="Token Icons" />
+          <Chip label="Unlimited Metadata" />
+          <Chip label="Updatable" />
+          <Chip label="Extensible" />
+          <Chip label="Notifying" />
+          <Chip label="Unified" />
+        </div>
+      </td>
+      <td>
+        <div className="chip-container">
+          <Chip label="Authentic" />
+          <Chip label="Safer" />
+          <Chip label="Unlimited Metadata" />
+          <Chip label="Updatable" />
+          <Chip label="Better IDs" />
+          <Chip label="Iteratable" />
+          <Chip label="Batch Transfers" />
+        </div>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+:::success
+Learn more about the LSPs 🧱
+To learn more about the main building blocks of LUKSO check out the [LUKSO Standard Proposals (**LSPs**)](../standards/introduction.md) page.
+:::
+
+## How do I start?
+
+<div>
+  <CallToActionButton
+    style={{
+      display: 'inline-block !important',
+    }}
+    icon="material-symbols:counter-1"
+    color="white"
+    target="_blank"
+    link="https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn"
+    text="Install the Universal Profile Browser Extension 🧩"
+  />
+
+  <div
+    style={{
+      display: 'flex',
+      justifyContent: 'center',
+      fontSize: '3em',
+    }}
+  >
+    <p>⬇</p>
+  </div>
+
+  <CallToActionButton
+    icon="material-symbols:counter-2"
+    color="white"
+    target="_blank"
+    link="https://my.universalprofile.cloud"
+    text="Create your Universal Profile 🆙"
+  />
+
+  <div
+    style={{
+      display: 'flex',
+      justifyContent: 'center',
+      fontSize: '3em',
+    }}
+  >
+    <p>⬇</p>
+  </div>
+</div>
+
+<CallToActionButton
+  icon="material-symbols:counter-3"
+  color="white"
+  target="_blank"
+  link="https://www.lukso.network/ecosystem"
+  text="Explore the Ecosystem 🌐"
+/>
+
+## Ready to build?
+
+<CardWithImage CardData={CardData} />
```

Comment:
The component name make it sounds like it's only one card, but is it not multiple cards?

```suggestion
<CardsWithImage CardData={CardData} />
```